contrib_bots/lib/hangman.py

```diff
@@ -0,0 +1,142 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
I got a travis error when I didnt have it there. But now that I think about it, that travis error was probably from when I had 1 print in my code from testing. I think I can remove that.